design-library/src/components/BccInfo/BccInfo.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+import type { Component } from "vue";
+type Props = {
+  title: string;
+  infoIcon?: Component;
```

Comment:
The type we usually use for icons is `string | Component | Function` - let's keep it the same for all components (whether we need string is debatable but `Function` is useful in some cases).